client/src/test/java/org/spine3/base/IdentifierShould.java

```diff
@@ -26,11 +26,11 @@
 import com.google.protobuf.UInt32Value;
 import com.google.protobuf.UInt64Value;
 import org.junit.Test;
-import org.spine3.base.Identifier.Type;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
+import static org.spine3.base.Identifier.*;
```

Comment:
I suspect we don't have wildcard imports in our codebase, do we?
Codacy should've caught that, but it does not check the test codebase.

Please address.